pulp_2_tests/tests/docker/api_v2/test_copy.py

```diff
@@ -124,6 +124,62 @@ def test_02_copy_tags(self):
             repo['content_unit_counts'].get('docker_tag', 0),
         )
 
+    @skip_if(bool, 'repo', False)
+    def test_02_copy_tags_user_metadata(self):
+        """Copy tags with user_metadata from one repository to another.
+
+        Assert the user metadata associated with a tag is present in
+        both repositories.
+
+        Steps:
+
+        1. Add user metadata to the first tag in the source repo.
+        2. Copy the tags from one repo to the other.
+        3. Verify that the user_metadata is copied to the other repo.
+
+        This test targets the following
+
+        * `Pulp #3242 <https://pulp.plan.io/issues/3242>`_.
+        * `Pulp-2-tests #72 <https://github.com/PulpQE/Pulp-2-Tests/issues/72>`_.
+        """
+        if not selectors.bug_is_fixed(3892, self.cfg.pulp_version):
+            self.skipTest('https://pulp.plan.io/issues/3892')
+        client = api.Client(self.cfg, api.json_handler)
+
+        # Step 1
+        tag_first_repo = search_units(
+            self.cfg, self.repo,
+            {'type_ids': ['docker_tag']}
+        )[0]
+
+        user_metadata = {
+            'dummy_key_1': 'dummy_value_1',
+            'dummy_key_2': 'dummy_value_2',
+        }
+        type(self)._set_user_metadata(client, tag_first_repo, user_metadata)
```

Comment:
Perhaps, we can just make a simple call  `_set_user_metadata`. This method does not return any data.